docs/source/user-guide/nlp/quickstarts/README.rst

```diff
@@ -0,0 +1,12 @@
+NLP Quickstarts
+===============
+
+Deepchecks for NLP is currently in alpha stage. The following quickstart will walk you through how you can try the
+existing functionality quickly for your own NLP project.
+
+In addition to the individual quickstarts available here, you can also use built-in
+:ref:`Suites <deepchecks_hierarchy_suite>` to run all the checks relevant to a specific part in your ML pipeline.
+For the API reference of each of the built-in Suites, check out :mod:`deepchecks.nlp.suites`.
```

Comment:
How about adding a link to our community slack and inviting users to give us feedback?